src/components/SidebarButtonWithBadge.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import SidebarButtonWithBadge from './SidebarButtonWithBadge';
+import { cleanup } from '@testing-library/react';
+
+afterEach(cleanup);
+
+describe('<SidebarButtonWithBadge/>', () => {
+  it('renders without crashing', () => {
+    global.renderWithRedux(<SidebarButtonWithBadge />);
+  });
+
+  it('renders appropriate content', () => {
+    const { getByTestId } = global.renderWithRedux(
+      <SidebarButtonWithBadge showBadge={true} color="color" />
+    );
+    // check that badge is present
+    expect(getByTestId('SidebarButton-icon-button').lastChild.tagName).toBe(
```

Comment:
again, i don't really care if it's a div or not -- what's the thing you're actually trying to test?

Reply:
Done